modules/claims_api/config/schemas/526.json

```diff
@@ -1322,7 +1322,7 @@
       "example": false
     },
     "claimDate": {
-      "description": "Date when claim is being submitted to the VA. Date cannot be in the future.",
+      "description": "Date (or Timestamp with TimeZone) when claim is being submitted to the VA. Date cannot be in the future. UTC TimeZone is assumed unless otherwise specified.",
       "type": "string",
       "pattern": "^(\\d{4})-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1])$",
```

Comment:
Has this existing pattern been tested to see if it can handle a timestamp with time zone?
We should really add a spec that tests what happens when someone passes us a timestamp with a time zone.

Reply:
Is this an acceptable regex?
`^(?:[0-9]{4})-(?:0[1-9]|1[0-2])-(?:0[1-9]|[1-2][0-9]|3[0-1])(?:T[0-1][0-9]:[0-5][0-9]:[0-5][0-9](?:[-+][0-9]{2}:*[0-9]{2}| *[a-zA-Z]{3}))*$`